navigator/frontend/package.json

```diff
@@ -73,14 +73,18 @@
     "react-dom": "^15.6.2",
     "react-markdown": "2.5.0",
     "react-overlays": "1.0.0-beta.2",
-    "react-redux": "^5.0.5",
+    "react-redux": "5.0.6",
     "react-virtualized": "^9.9.0",
     "redux": "^3.7.0",
-    "redux-thunk": "^2.2.0",
+    "redux-thunk": "~2.2.0",
     "route-parser": "0.0.5",
     "styled-components": "2.1.0",
     "typescript-collections": "1.2.5",
     "uuidjs": "3.5.3",
     "whatwg-fetch": "^2.0.3"
+  },
+  "resolutions": {
+      "**/@types/react": "~16.0.0",
```

Comment:
The fact that we use `@types/react` 16.x but react 15.x seems super sketchy but that’s what we’ve been doing before as well. This doesn’t change anything in that regard.